data/harvesting.txt

```diff
@@ -253,3 +253,91 @@ outfit "Yottrite"
 	"mass" 3.14
 	"installable" -1
 	description "An extraordinarily rare crystalline formation, forged over tens of thousands of years in the molecular clouds around supernova remnants. Although less advanced species can potentially harvest it, very few civilizations other than the Quarg have developed the technology needed to incorporate Yottrite in an alloy."
+
+
+
+# The following are not traditional asteroids, and are not present at the start of the game.
+# They are added to specific systems in the Ember Waste during the events of the 'Remnant: Cognizance' missions.
+
+minable "plant"
+	sprite "asteroid/plant/plant"
+	hull 2800
+	payload "Void Orchid" 21
+	explode "smoke" 30
+	explode "bolide" 30
+	explode "final explosion medium"
+
+minable "plant2"
+	sprite "asteroid/plant2/plant2"
+	hull 2600
+	payload "Void Orchid" 20
+	explode "smoke" 30
+	explode "bolide" 30
+	explode "final explosion medium"
+
+minable "plant cluster"
+	sprite "asteroid/plant cluster/plant cluster"
+	hull 1200
+	payload "Void Orchid" 11
+	explode "smoke" 20
+	explode "bolide" 20
+	explode "final explosion small"
+
+minable "large plant"
+	sprite "asteroid/large plant/large plant"
+	hull 3800
+	payload "Void Orchid" 41
+	explode "smoke" 30
+	explode "bolide" 40
+	explode "final explosion medium"
+
+minable "large plant2"
+	sprite "asteroid/large plant2/large plant2"
+	hull 3600
+	payload "Void Orchid" 41
+	explode "smoke" 30
+	explode "bolide" 40
+	explode "final explosion medium"
+
+minable "large plant cluster"
+	sprite "asteroid/large plant cluster/large plant cluster"
+	hull 2200
+	payload "Void Orchid" 22
+	explode "smoke" 30
+	explode "bolide" 40
+	explode "final explosion medium"
+
+minable "live crystal"
+	sprite "asteroid/livecrystal/livecrystal"
+	hull 1000
+	payload "Void Orchid" 8
+	explode "smoke" 15
+	explode "bolide" 10
+	explode "final explosion small"
+
+minable "space flora"
+	sprite "asteroid/space flora/space flora"
+	hull 1000
+	payload "Void Orchid" 8
+	explode "smoke" 15
+	explode "bolide" 10
+	explode "final explosion small"
+
+minable "large space flora"
+	sprite "asteroid/large space flora/large space flora"
+	hull 2000
+	payload "Void Orchid" 16
+	explode "smoke" 25
+	explode "bolide" 20
+	explode "final explosion medium"
+
+
+outfit "Void Orchid"
+	plural "Void Orchids"
+	category "Special"
+	cost 15000
+	thumbnail "outfit/harvested silicon"
+	"flotsam sprite" "effect/flotsam silicon"
+	"mass" 1
+	"installable" -1
```

Comment:
```suggestion
	"installable" -1
	"minable" 1
```
With #6361 now merged, we'll need this attribute to make it properly show as `(Mined from asteroids)` on the map, along with when it's harvested.